defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
-bbb_hostname: "{{ ansible_fqdn }}"
```

Comment:
Why did you change all double quotes to single quotes?
Just curious.

Reply:
Honestly at first my prettier plugin changed to single quotes. but after I rechecked the file I saw that this file partially uses single quotes and partially double quotes e.g: https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/bbb/2.3/defaults/main.yml#L90-L94 and https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/bbb/2.3/defaults/main.yml#L124-L126. Because of that I thought it would make more sense to use single quotes instead. I can change it back if you want.

Reply:
I guess you could say it's flavor double quotes are need when you do something like this:
```
"'service_name' in groups"
```